MAESTRO/MAESTRO_PipeInit.py

```diff
@@ -176,37 +176,37 @@ def scrna_parser(subparsers):
 
     # Input files arguments
     group_input = workflow.add_argument_group("Input files arguments")
-    group_input.add_argument("--platform", dest = "platform", default = "10x-genomics", 
-        choices = ["10x-genomics", "Dropseq", "Smartseq2"], 
+    group_input.add_argument("--platform", dest = "platform", default = "10x-genomics",
+        choices = ["10x-genomics", "Dropseq", "Smartseq2"],
         help = "Platform of single cell RNA-seq. DEFAULT: 10x-genomics.")
-    group_input.add_argument("--fastq-dir", dest = "fastq_dir", type = str, required = True,  
+    group_input.add_argument("--fastq-dir", dest = "fastq_dir", type = str, required = True,
         help = "Directory where fastq files are stored")
-    group_input.add_argument("--fastq-prefix", dest = "fastq_prefix", type = str, default = "", 
+    group_input.add_argument("--fastq-prefix", dest = "fastq_prefix", type = str, default = "",
         help = "Sample name of fastq file, only for the platform of '10x-genomics'. "
         "If there is a file named pbmc_1k_v2_S1_L001_I1_001.fastq.gz, the prefix is 'pbmc_1k_v2'.")
-    group_input.add_argument("--fastq-barcode", dest = "fastq_barcode", type = str, default = "", 
+    group_input.add_argument("--fastq-barcode", dest = "fastq_barcode", type = str, default = "",
         help = "Specify the barcode fastq file, only for the platform of 'Dropseq'. "
         "If there are multiple pairs of fastq, please provide a comma-separated list of barcode fastq files. "
         "For example, --fastq-barcode test1_1.fastq,test2_1.fastq")
-    group_input.add_argument("--fastq-transcript", dest = "fastq_transcript", type = str, default = "", 
+    group_input.add_argument("--fastq-transcript", dest = "fastq_transcript", type = str, default = "",
         help = "Specify the transcript fastq file, only for the platform of 'Dropseq'. "
         "If there are multiple pairs of fastq, please provide a comma-separated list of barcode fastq files. "
         "For example, --fastq-barcode test1_2.fastq,test2_2.fastq")
     group_input.add_argument("--species", dest = "species", default = "GRCh38",
-        choices = ["GRCh38", "GRCm38"], type = str, 
+        choices = ["GRCh38", "GRCm38"], type = str,
         help = "Specify the genome assembly (GRCh38 for human and GRCm38 for mouse). DEFAULT: GRCh38.")
 
     # Output arguments
     group_output = workflow.add_argument_group("Running and output arguments")
-    group_output.add_argument("--cores", dest = "cores", default = 8, 
-        type = int, help = "The number of cores to use. DEFAULT: 8.")
-    group_output.add_argument("--rseqc", dest = "rseqc", action = "store_true", 
+    group_output.add_argument("--cores", dest = "cores", default = 10,
```

Comment:
10 cores is the best for lisa right? I remember Allen was saying

Reply:
Yes. So I changed the default to 10.